src/app/core/component/cardrenderer/card-renderer.component.ts

```diff
@@ -4,33 +4,105 @@ import {ActivatedRoute, Router} from '@angular/router';
 import UriEntity from '../../model/hal/uri-entity.model';
 import Pattern from '../../model/hal/pattern.model';
 import {HalLink} from '../../model/hal/hal-link.interface';
+import {PatternService} from '../../service/pattern.service';
 
 @Component({
-    selector: 'pp-card-renderer',
-    templateUrl: './card-renderer.component.html',
-    styleUrls: ['./card-renderer.component.scss']
+  selector: 'pp-card-renderer',
+  templateUrl: './card-renderer.component.html',
+  styleUrls: ['./card-renderer.component.scss']
 })
 export class CardRendererComponent {
 
-    @Input() uriEntities: Array<Pattern>;
-    @Input() showLinks = true;
+  @Input() uriEntities: Array<Pattern>;
+  @Input() showLinks = true;
     @Output() createEntityClicked: EventEmitter<void> = new EventEmitter<void>();
 
-    constructor(private zone: NgZone,
-                private router: Router,
-                private activatedRoute: ActivatedRoute) {
-    }
+  constructor(private zone: NgZone,
+              private router: Router,
+              private activatedRoute: ActivatedRoute,
+              private patternService: PatternService) {
+  }
 
-    navigate(pattern: UriEntity): void {
-        this.zone.run(() => {
-            this.router.navigate([UriConverter.doubleEncodeUri(pattern.id)], {relativeTo: this.activatedRoute});
-        });
-    }
+  navigate(pattern: UriEntity): void {
+    this.zone.run(() => {
+      this.router.navigate([UriConverter.doubleEncodeUri(pattern.id)], {relativeTo: this.activatedRoute});
+    });
+  }
 
-    getLinkCount(directedEdges: HalLink[] | HalLink): number {
-        if (!directedEdges) {
-            return 0;
+  delete(pattern: Pattern): void {
+    this.patternService.deletePattern(pattern._links.self.href)
+      .subscribe(
+        value => this.handlePatternDelete(pattern),
+        error => {
+          console.log(`Delete unsuccessful! ${error}`);
         }
```

Comment:
can you add  a toaster notification after the (un)successful completion of the action?